### PR TITLE
Redirect HTML docs to the repo

### PR DESCRIPTION
**Pull request by mikemckiernan**
Redirect from HTML GitHub pages to the repo.

### CODE REVIEW

File: index.html
Comment (dglogo) [inline index.html:1]: Would love to see a quick demo if possible either live or via a quick video. thanks!

Comment (mikemckiernan) [reply index.html:1]: Daniel, I'm so sorry that I missed this message.  I just made a post in the team Slack channel so you can check a few URLs on my fork.

**Comment by dglogo** [comment]
@mikemckiernan please update the html redirection such that, for example, https://mikemckiernan.github.io/GenerativeAIExamples/latest/vector-database.html
redirects to the respective doc in github (https://gitlab-master.nvidia.com/chat-labs/OpenSource/GenerativeAIExamples/-/blob/main/docs/vector-database.md)?

Can we please do this for all the links in the old docs?

**Comment by mikemckiernan** [comment]
> @mikemckiernan please update the html redirection such that, for example, https://mikemckiernan.github.io/GenerativeAIExamples/latest/vector-database.html redirects to the respective doc in github (https://gitlab-master.nvidia.com/chat-labs/OpenSource/GenerativeAIExamples/-/blob/main/docs/vector-database.md)?
> 
> Can we please do this for all the links in the old docs?

I do not believe there is a one-to-one correlation from the HTML docs to the Markdown documents.  Customers will see the nav in the README and reorient themselves quickly, I think.

**Review by dglogo — APPROVED** [review]
Approved, but requires to fast follow updates. Thanks

**Pull request merged**